Makefile

```diff
@@ -134,3 +134,9 @@ proto: $(BUF) $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_GRPC) $(PROTO_TEST_DIR)/test.prot
 		--go_out=$(PROTO_TEST_DIR)/../ \
 		--go-grpc_out=$(PROTO_TEST_DIR)/../ \
 	    $(PROTO_TEST_DIR)/*.proto
+
+.PHONY: buf.gen
+buf.gen:
+	@go run github.com/bufbuild/buf/cmd/buf@v1.26.1 generate \
+           --template ./testing/testvalidate/testvalidate.buf.gen.yaml \
+           --path ./testing/testvalidate/v1
```

Comment:
On second thought, we actually already have `buf` available as `$(BUF)`. I think we probably need to update the version used though. Could you try using `$(BUF)`? If it doesn't work please upgrade the version we're using. I'm happy to help with this.

Reply:
I've replaced go run ... with $(BUF) + upgrade buf to v1.26.1.

Probably it will fix also lint step in CI

Reply:
Hmm, lint step still failing, but now says that there's no license header in `*.pb.go` files. Can you please take a look and help me a little bit?

Reply:
Hm, yes, let me see

Reply:
That should do it